test/Pomelo.EntityFrameworkCore.MySql.Tests/MysqlRelationalConnectionTest.cs

```diff
@@ -30,7 +26,7 @@ public void Can_create_master_connection_string()
             {
                 using (var master = connection.CreateMasterConnection())
                 {
-                    Assert.Equal(@"server=localhost;port=3306;database=mysql;user id=root;password=Password12!;allowuservariables=True;pooling=False", master.ConnectionString);
+                    Assert.Equal(@"Server=localhost;Port=3306;Database=mysql;User Id=root;Password=Password12!;AllowUserVariables=True;Pooling=False", master.ConnectionString);
```

Comment:
We doesn't require AllowUserVariables now. In other words, we can remove AllowUserVariables in this version.